apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -117,14 +119,19 @@ function DeleteDialog({
   eventTypeId,
   open,
   onOpenChange,
-}: { isManagedEvent: string; eventTypeId: number } & Pick<DialogProps, "open" | "onOpenChange">) {
+  isEventTypeDeleted,
```

Comment:
`isEventTypeDeleted` is a reference that is being updated in this function. This isn't a good coding practice. I would recommend adding a `onDelete` prop instead.